ontology/mamo.md

```diff
@@ -21,6 +21,7 @@ license:
   label: Artistic License 2.0
 activity_status: active
 preferredPrefix: MAMO
+domain: simulation
```

Comment:
@gambardella is no longer on GitHub, see #1886 for suggesting this is marked as orphaned